### PR TITLE
[ASM] EXPANDR-9134 service account changes

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9134
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8417

## Description
Add support for distinguishing service accounts for IAM enrichment steps

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@capanw , The PR looks good.  Adding a more broad playbook question in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9134

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml [hunk 1]
```diff
@@ -1106,7 +1106,7 @@ tasks:
                 value:
                   simple: email
       val3:
-        simple: GCP
+        simple: GCP-SERVICE-ACCOUNT
```
[the remark this answers is not in the document]
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml]: added accounttype field and updated RankServiceOwners script

File: Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_GCP_Enrichment.png
[the remark this answers is not in the document]
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_GCP_Enrichment.png:1]: Done!!

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
made another comment in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9134

**Comment by capanw** [comment]
> made another comment in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9134

@johnnywilkes, added a new branch in the playbook

**Review by johnnywilkes — APPROVED** [review]
@capanw , thanks for making changes
@BigEasyJ , what you incorporated looks good

**Comment by capanw** [comment]
@maimorag, this is approved from RC side. Please review and merge when you have time. Thanks

**Review by maimorag — APPROVED** [review]

**Review by maimorag — APPROVED** [review]
Hi @capanwת
Good work :)

**Pull request merged**